install_files/securedrop-ossec-agent/DEBIAN/control

```diff
@@ -4,7 +4,7 @@ Priority: optional
 Maintainer: SecureDrop Team <securedrop@freedom.press>
 Homepage: https://freedom.press/securedrop
 Package: securedrop-ossec-agent
-Version: 2.8.1+0.3.3
+Version: 2.8.2+0.3.3
```

Comment:
This should be `2.8.2+0.3.4` since you are asking to merge it into the release/0.3.4 branch.